packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,400 @@
+---
+title: Design Guidelines for Paste
+description: Notes on how to use Paste design assets
```

Comment:
We don't use the term "assets" anywhere else in Paste. Is this a guide for how to use the assets? Or how to get started as a designer with installing and using Paste?